editor/src/components/editor/action-types.ts

```diff
@@ -287,6 +287,19 @@ export type ToggleInterfaceDesignerCodeEditor = {
   action: 'TOGGLE_INTERFACEDESIGNER_CODEEDITOR'
 }
 
+export type IncrementCodeEditorFontSize = {
+  action: 'INCREMENT_CODE_EDITOR_FONT_SIZE'
+}
+
+export type DecrementCodeEditorFontSize = {
+  action: 'DECREMENT_CODE_EDITOR_FONT_SIZE'
+}
+
+export type UpdateCodeEditorFontSize = {
```

Comment:
Since we always know the current font size when dispatching these actions, we can remove the `Increment` and `Decrement` actions and just use the `Update` action in all cases.